scripts/firedrake-install

```diff
@@ -35,6 +35,10 @@ travis = "TRAVIS" in os.environ
 travis_testing_firedrake = "FIREDRAKE_TRAVIS_TESTS" in os.environ
 travis_cache_dir = os.environ.get("CACHE_DIRECTORY", None)
 
+# More generic hack for drone.io CI
+ci = "CI" in os.environ
+ci_event = os.environ["CI_EVENT"]
```

Comment:
This raises an exception if CI_EVENT is not set. You probably want something like:

     os.environ.get("CI_EVENT", default=None)